src/openms/source/CHEMISTRY/ISOTOPEDISTRIBUTION/IsoSpecWrapper.cpp

```diff
@@ -214,6 +239,8 @@ namespace OpenMS
     return ID;
   }
 
+  IsoSpecThresholdWrapper::~IsoSpecThresholdWrapper() = default;
```

Comment:
Today I learned that in case of these partial types we also need to put the default destructor into the cpp file, In header it fails.